src/theme/variables/edgeDark.js

```diff
@@ -216,6 +216,8 @@ export const edgeDark: Theme = {
   tabBarTopOutlineColors: [palette.edgeNavy, palette.edgeNavy],
   tabBarIcon: palette.white,
   tabBarIconHighlighted: palette.edgeMint,
+  extraTabBarIconFont: '',
+  extraTabBarIconName: '',
```

Comment:
Couldn't this just go into the config, as opposed to the theme? This does not change dynamically.